browser/src/DataPage/DataPage.tsx

```diff
@@ -63,6 +63,20 @@ const DataPage = () => {
       </TableOfContentsSection>
 
       <TextSection>
+        <div>
+          <Badge level="warning">Warning</Badge> Access to gnomAD data in{' '}
+          <ExternalLink href="https://learn.microsoft.com/en-us/azure/open-datasets/dataset-gnomad">
+            Azure Open Datasets
+          </ExternalLink>{' '}
+          will be deprecated on August 15. Consider transitioning to accessing gnomAD data in{' '}
+          <ExternalLink href="https://cloud.google.com/public-datasets">
+            Google Cloud Public Datasets
+          </ExternalLink>{' '}
+          or the{' '}
+          <ExternalLink href="https://registry.opendata.aws/">
+            Registry of Open Data on AWS
```

Comment:
```suggestion
            Registry of Open Data on AWS.
```
sorry, I missed this in my suggestion!